sdks/python/apache_beam/task/task_worker/k8s/__init__.py

```diff
@@ -0,0 +1,18 @@
+#
```

Comment:
I think I would rename `apache_beam.task.task_worker.k8s` to `apache_beam.task_workers.kubejob`